Procfile

```diff
@@ -1 +1 @@
-web: node server.js
+web: npm start
```

Comment:
@shawnbot that's a good thing to flag.  I am not too sure about running `nodemon` on production either since it is marketed as a development tool. and from what I found also seems to suggest folks are using `pm2` as a way of managing production servers. 

to me sounds like there are two things we want to decide on: 
* what we want to use to serve up the app, and
* how we handling crashes 

let's chat more on Slack and also suggested as a topic for our next Engineering meeting.